network/network.go

```diff
@@ -100,6 +101,11 @@ type Network interface {
 
 	// Process returns the network's Process
 	Process() goprocess.Process
+
+	// EventBus returns the network's Event Bus
+	// we can subscribe to this bus to listen for connection/disconnection of peers,
+	// opening/closing of streams etc etc
+	EventBus() event.Bus
```

Comment:
Do we need to _expose_ this on the swarm? We can probably just pass it in to the constructor.